apps/frontend-manage/src/components/evaluation/TableChart.tsx

```diff
@@ -25,27 +25,23 @@ function TableChart({
 }: TableChartProps): React.ReactElement {
   const ref = useRef<{ reset: () => void }>(null)
 
+  const formatCorrect = (value: boolean | undefined | null): string => {
+    if (value === true) return 'T'
+    if (value === false) return 'F'
+    return '--'
+  }
+
   const tableData = useMemo(() => {
     if (QUESTION_GROUPS.CHOICES.includes(data.questionData.type)) {
       return (data.questionData as ChoicesQuestionData).options.choices.map(
         (choice: Choice, index: number) => {
           return {
             count: data.results[index].count,
-            value: (
-              <Markdown
-                content={choice.value}
-                className={{ img: 'max-h-32' }}
-              />
-            ),
-            correct: choice.correct ? 'T' : 'F',
+            value: choice.value,
+            correct: formatCorrect(choice.correct),
```

Comment:
why is the formatting done on this level and not passed through as a formatter for the correct column? now that we have this abstraction? :)

Reply:
this wasn't possible until now, because of type constraints that were only lifted with the most recent design-system release. Will be changed in this PR

Reply:
done